tests/runner/src/env.rs

```diff
@@ -190,7 +190,7 @@ impl Env {
                     "start".to_string(),
                     "-c".to_string(),
                     self.generate_config_file(subcommand, db_ctx),
-                    "--http-addr=127.0.0.1:5001".to_string(),
+                    "--http-addr=127.0.0.1:5002".to_string(),
```

Comment:
Do not expect to change this port, what's the problem here?

Reply:
kind registry service default use 5001